pages.th/windows/cls.md

```diff
@@ -0,0 +1,8 @@
+# cls
+
+> ล้างตัวอักษรทั้งหมดที่อยู่บนหน้าต่าง
+> ข้อมูเพิ่มเติ่ม: https://docs.microsoft.com/windows-server/administration/windows-commands/cls
```

Comment:
```suggestion
> ข้อมูเพิ่มเติ่ม: https://learn.microsoft.com/windows-server/administration/windows-commands/cls
```
Reference: #8543, There is an ongoing PR to update all Microsoft links to `learn.microsoft.com`. The original page will be updated soon.